go.mod

```diff
@@ -3,8 +3,9 @@ module github.com/nuts-foundation/nuts-crypto
 go 1.14
 
 require (
-	github.com/deepmap/oapi-codegen v1.3.12
+	github.com/deepmap/oapi-codegen v1.3.7
```

Comment:
What about 1.3.13? Is that working OK?

Reply:
Didn't work either, since after reinstalling Macbook I had v13 in the first place so I thought that was the culprit, however downgrading to v12 it didn't fix it. Still strange because I recently added another API call and regenerated the code so it's still worth deepdiving into it to find out what's wrong. Also because we currently can't upgrade which might be a security issue in the future.